install/image-references

```diff
@@ -18,3 +18,7 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0
+  - name: baremetal-machine-controllers
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-baremetal-machine-controllers:v4.0.0
```

Comment:
```suggestion
      name: quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0
```
New images are no longer mirrored to docker.io

Reply:
Thanks, I completed `s/docker/quay/` in the patch